.github/workflows/scripts/reclaim_disk_space.sh

```diff
@@ -1,20 +1,16 @@
-#!/bin/sh
+#!/bin/sh -x
 
 set -eu
 
 # remove 4GiB of images
 sudo systemd-run docker system prune --force --all --volumes
 
+# remove unused packages
+sudo apt remove -q --purge firefox
+
 # remove unused software
 sudo systemd-run rm -rf \
   "$AGENT_TOOLSDIRECTORY" \
   /opt/* \
   /usr/local/* \
-  /usr/share/az* \
-  /usr/share/dotnet \
```

Comment:
I haven't run enough iterations to be certain, but I dropped `/usr/share/dotnet` because I've seen `apt update` complain about that package.  Since it seem we had enough space to run all the tests let's play it safe for now and drop it.

Reply:
Actually, I see the 22.04 builder did time out.  Let me put that back since we may have run out of space.